src/frame/initialization/index.ts

```diff
@@ -73,9 +77,9 @@ function handleVerifyResult(
  * @param {CallbackToPromise} jwtCallback                  Method which when invoked will return a Promise which will be resolved with a JWT token
  * @param {string}            deviceAndSigningSymmetricKey Optional symmetric key that is used to decrypt the users device and signing keys
  */
-export function initialize(jwtToken: string, deviceAndSigningSymmetricKey?: string) {
+export const initialize = (jwtToken: string, deviceAndSigningSymmetricKey?: string) => {
```

Comment:
Can drop the `{ return }` now too since it's a single sequence. Same comment in a few places for this file.